packages/react-ai/src/components/AIConversation/AIConversation.tsx

```diff
@@ -0,0 +1,82 @@
+import * as React from 'react';
+import { Flex, ScrollView, Text, TextProps } from '@aws-amplify/ui-react';
+import {
+  IconAssistant,
+  IconUser,
+  useIcons,
+} from '@aws-amplify/ui-react/internal';
+import { createAIConversation } from './createAIConversation';
+import { AIConversationInput, AIConversationProps, Avatars } from './types';
+import { MessagesControl } from './views/Controls/MessagesControl';
+import { FieldControl } from './views';
+import { MessageList } from './views/default/MessageList';
+import { Form } from './views/default/Form';
+import { PromptList } from './views/default/PromptList';
+import { AutoHidablePromptControl } from './views/Controls';
+import { ComponentClassName } from '@aws-amplify/ui';
+
+interface _AIConversationProps
+  extends AIConversationProps,
+    AIConversationInput<{}> {}
+
+export const AIConversation = ({
+  messages,
+  handleSendMessage,
+  avatars,
+  controls,
+  responseComponents,
+  suggestedPrompts,
+  variant,
+}: _AIConversationProps): JSX.Element => {
+  const icons = useIcons('aiConversation');
+  const defaultAvatars: Avatars = {
+    ai: {
+      username: 'Assistant',
+      avatar: icons?.assistant ?? <IconAssistant />,
+    },
+    user: {
+      username: 'User',
+      avatar: icons?.user ?? <IconUser />,
+    },
+  };
+
+  const { AIConversation } = createAIConversation({
```

Comment:
Do we need to create the full `createAIConversation`? Is it easier if we just call the `createProvider` isntead?